packages/engine/lib/orchestrator/src/experiment.rs

```diff
@@ -157,7 +158,26 @@ impl Experiment {
                 }
                 proto::EngineStatus::SimStatus(status) => {
                     debug!("Got simulation run status: {status:?}");
-                    // TODO: OS - handle status fields
+                    for stop_message in status.stop_msg {
+                        let reason = if let Some(reason) = stop_message.reason.as_ref() {
+                            format!(": {reason}")
+                        } else {
+                            String::new()
+                        };
+                        match stop_message.status {
+                            StopStatus::Success => {
+                                tracing::info!("Simulation stopped sucessfully{reason}");
```

Comment:
```suggestion
                                tracing::info!("Simulation stopped successfully{reason}");
```

Reply:
Also, I wonder whether we should log that an agent stopped the simulation (as opposed to it stopping in some other way) or just regard that as an implementation detail.

Reply:
May we should add the agent's UUID?